src/training/train.py

```diff
@@ -35,11 +34,8 @@ def train_one_epoch(model_video, model_text, logit_scale, data, epoch, optimizer
 
         optimizer.zero_grad()
 
-        video_embeddings = model_video(embeddings)
-        with torch.no_grad():
-            text_embeddings = model_text(toks).float()
-
-        loss = loss_func(video_embeddings, text_embeddings, logit_scale.exp())
+        video_embeddings, text_embeddings, logit_scale  = model_video(embeddings, toks, prenorm=False, postnorm=False)
```

Comment:
I think we do want both prenorm and postnorm for training.

1. We want the model to learn the normalized CLIP embedding distribution
2. Why not normalize output if that's the function we want to approximate in the first place